src/psyclone/tests/psyir/frontend/fparser2_test.py

```diff
@@ -42,14 +42,15 @@
 from psyclone.psyGen import PSyFactory, Node, Directive, Schedule, \
     CodeBlock, Assignment, Return, UnaryOperation, BinaryOperation, \
     NaryOperation, Literal, IfBlock, Reference, Array, KernelSchedule, \
-    Symbol, SymbolTable, \
+    Symbol, SymbolTable, Container, \
     InternalError, GenerationError
 from psyclone.psyir.frontend.fparser2 import Fparser2Reader
 from psyclone.dynamo0p3 import DynKern, DynKernMetadata
-
+from fparser.common.readfortran import FortranStringReader
```

Comment:
Can you solve this 3 pylint issues:

- [unused-import] Unused api imported from fparser as fpapi
- [unused-import] Unused DynKern imported from psyclone.dynamo0p3
- [wrong-import-order] third party import "from fparser.common.readfortran import FortranStringReader" should be placed before "from psyclone.psyGen ..."

Reply:
And FortranStringReader is re-declared inside some tests.

Reply:
FIxed. There were a lot of re-declarations to remove!